Composer/packages/extensions/obiformeditor/src/Form/widgets/ExpressionWidget.tsx

```diff
@@ -62,7 +64,7 @@ export const ExpressionWidget: React.FC<ExpresionWidgetProps> = props => {
 
   return (
     <>
-      <WidgetLabel label={label} description={description} id={id} />
+      {label && <WidgetLabel label={label} description={description} id={id} />}
```

Comment:
```suggestion
      {!!label && <WidgetLabel label={label} description={description} id={id} />}
```

Because `undefined` will become `'undefined' && <WidgetLabel ...>`.